src/main/java/uk/gov/hmcts/darts/common/entity/Courthouse.java

```diff
@@ -24,31 +22,17 @@ public class Courthouse {
     @GeneratedValue(strategy = GenerationType.IDENTITY)
```

Comment:
Please can you update the annotations to have `GeneratedValue` and `SequenceGenerator` using the SEQUENCE `moj_crt_seq` we should be adding into the flyway script